### DISCUSSION TITLE
Deploy Copy

### DISCUSSION THREAD

**Discussion by yuri-kiss** [TurboWarp/extensions#574] [Q&A] [answered]
How would you deploy your own copy of this repo, I have tried but it kept giving build error.

**Comment by GarboMuffin** [comment]
perhaps you should post the error you get

**Comment by yuri-kiss** [comment, marked as answer]
I solved it, I just deployed the page with github actions and made a custom workflow that was copied from the actual deploy.yml file and changed the branch from master to main, and that seemed to fix it

**Reply by yuri-kiss** [reply]
also deployed it with the custom workflow
